src/ResourceManager/ServiceBus/ChangeLog.md

```diff
@@ -20,6 +20,157 @@
+   - 'Remove-AzureRmServiceBusTopicAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Remove-AzureRmServiceBusAuthorizationRule'
+   
+   **Set-AzureRmServiceBusTopicAuthorizationRule**,
+   - 'Set-AzureRmServiceBusTopicAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Set-AzureRmServiceBusAuthorizationRule'
+   
+   **New-AzureRmServiceBusNamespaceKey**, 
+   - 'New-AzureRmServiceBusNamespaceKey' cmdlet is removed in current build. Please use the New cmdlet 'New-AzureRmServiceBusKey'
+   
+   **Get-AzureRmServiceBusQueueAuthorizationRule**, 
+   - 'Get-AzureRmServiceBusQueueAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Get-AzureRmServiceBusAuthorizationRule'
+   
+   **Get-AzureRmServiceBusQueueKey**,
+   - 'Get-AzureRmServiceBusQueueKey' cmdlet is removed in current build. Please use the New cmdlet 'Get-AzureRmServiceBusKey'
+   
+   **New-AzureRmServiceBusQueueAuthorizationRule**, 
+   - 'New-AzureRmServiceBusQueueAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'New-AzureRmServiceBusAuthorizationRule'
+   
+   **New-AzureRmServiceBusQueueKey**,
+   - 'New-AzureRmServiceBusQueueKey' cmdlet is removed in current build. Please use the New cmdlet 'New-AzureRmServiceBusKey'
+   
+   **Remove-AzureRmServiceBusQueueAuthorizationRule**,
+   - 'Remove-AzureRmServiceBusQueueAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'GRemove-AzureRmServiceBusAuthorizationRule'
+   
+   **Set-AzureRmServiceBusQueueAuthorizationRule**, 
+   - 'Set-AzureRmServiceBusQueueAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Set-AzureRmServiceBusAuthorizationRule'
+   
+   **Get-AzureRmServiceBusNamespaceAuthorizationRule**, 
+   - 'Get-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Get-AzureRmServiceBusAuthorizationRule'
+   
+   **Get-AzureRmServiceBusNamespaceKey**, 
+   - 'Get-AzureRmServiceBusNamespaceKey' cmdlet is removed in current build. Please use the New cmdlet 'Get-AzureRmServiceBusKey'
+   
+   **New-AzureRmServiceBusNamespaceAuthorizationRule**,
+   - 'New-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'New-AzureRmServiceBusAuthorizationRule'
+   
+   **Remove-AzureRmServiceBusNamespaceAuthorizationRule**,
+   - 'Remove-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Remove-AzureRmServiceBusAuthorizationRule'
+   
+   **Set-AzureRmServiceBusNamespaceAuthorizationRule**
+   - 'Set-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Set-AzureRmServiceBusAuthorizationRule'
+   
+   
+   The following properties are removed in this release:
+   
+   ** NamespceAttributes **
+   - Status
+   - Enabled
+   
+   ```powershell
```

Comment:
fix formatting (check the preview to make sure the formatting looks correct)

Reply:
I fixed this for you - the markdown file requires special formatting to display correctly.  Let me know if you have a problem with these changes.